### PR TITLE
Add docs for the stream processing platform Heron

### PR DESCRIPTION
**Pull request by ajorgensen**
You can find more information about the Heron project [here](https://github.com/twitter/heron)

### CODE REVIEW

File: docsets/Heron/docset.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "archive": "Heron.tgz",
+    "author": {
+        "name": "Andrew Jorgensen",
+        "link": "https://twitter.com/ajorgensen"
+    },
+    "aliases": [],
+    "specific_versions": [
+        { 
+            "version": "0.14.0",
+            "archive": "versions/0.14.0/heron.tgz"
```
Comment (Kapeli) [inline docsets/Heron/docset.json]: The archive you added is called `Heron.tgz`, not `heron.tgz` (case sensitivity).

File: docsets/Heron/docset.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Heron",
+    "version": "0.14.0",
+    "archive": "Heron.tgz",
```
Comment (Kapeli) [inline docsets/Heron/docset.json]: You need to add this archive as well. It's not enough to have it in the `versions/` folder only.

**Comment by Kapeli** [comment]
Also: delete the Sample_Docset.tgz.

**Comment by ajorgensen** [comment]
Thanks for the comments!

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**